Modules/Control/Control.py

```diff
@@ -23,7 +23,7 @@ def SetControl(self,mode=1):
 ''''
 returns the ratio of y1 to x1 where (x1;y1) is the vertical 2D vector to (x;y) vector
 '''
-def TwoDimensionsVerticalVector(x,y):#delete this fucntion
+def TwoDimensionsVerticalVector(x:float,y:float):#delete this fucntion
```

Comment:
If you really wish to delete it, delete it. Don't leave "delete me" comments in the master branch.
Otherwise, just change the name to `two_dim....` you know what I want...